touchforms/backend/xformplayer.py

```diff
@@ -478,6 +489,10 @@ def back_event (self):
         return self._parse_current_event()
 
     def answer_question (self, answer, _ix=None):
+        if self.use_api:
```

Comment:
do we need to do this for every call? i think a better pattern might be to make a base XFormSession class and then have a child class use the new api and another child class use the old way. that way there is less `if` branching in random parts of the code